2019-04-23 23:46:29,216 ERROR: Exception on /login [GET] [in /Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/flask/app.py:1761]
Traceback (most recent call last):
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/suprapto/try_flask/app/routes.py", line 65, in login
    return render_template('login.html', title='Sign In', form=form)
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/flask/templating.py", line 135, in render_template
    context, ctx.app)
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/flask/templating.py", line 117, in _render
    rv = template.render(context)
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/Users/suprapto/try_flask/app/templates/base.html", line 45, in template
    {% endwidth %}
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/environment.py", line 1005, in render
    return concat(self.root_render_func(self.new_context(vars)))
  File "/Users/suprapto/try_flask/app/templates/login.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/Users/suprapto/try_flask/app/templates/base.html", line 45, in template
    {% endwidth %}
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/parser.py", line 243, in parse_with
    drop_needle=True)
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/parser.py", line 144, in parse_statement
    self.fail_unknown_tag(token.value, token.lineno)
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/parser.py", line 97, in fail_unknown_tag
    return self._fail_ut_eof(name, self._end_token_stack, lineno)
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/parser.py", line 90, in _fail_ut_eof
    self.fail(' '.join(message), lineno)
  File "/Users/suprapto/try_flask/flaskenv/lib/python3.5/site-packages/jinja2/parser.py", line 59, in fail
    raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endwidth'. Jinja was looking for the following tags: 'endwith'. The innermost block that needs to be closed is 'with'.
2019-04-23 23:48:18,180 INFO: Prapsky Blog Startup [in /Users/suprapto/try_flask/app/__init__.py:48]
